packages/gasket-plugin-swagger/package.json

```diff
@@ -39,8 +39,10 @@
   },
   "homepage": "https://github.com/godaddy/gasket/tree/main/packages/gasket-plugin-swagger",
   "dependencies": {
+    "@fastify/swagger": "^6.0.0",
     "@types/swagger-ui-express": "^4.1.3",
     "express": "^4.16.3",
+    "fastify": "^4.1.0",
```

Comment:
`fastify` seems to only be used in your tests, and thus should be in `devDependencies`. It seems odd that `express` is not a dev dependency also, could it be moved, too?

Reply:
yeah, I don't think I see express used anywhere in the package